app/jobs/user_subscribe_job.rb

```diff
@@ -0,0 +1,9 @@
+class UserSubscribeJob < ActiveJob::Base
+  queue_as :default
+
+  def perform(name, email)
+    klass = Rails.env.test? ? FakeMailChimpClient : MailChimpClient
+    @client = klass.new
```

Comment:
@TheDudeWithTheThing would it make sense to put:

```
klass = Rails.env.test? ? FakeMailChimpClient : MailChimpClient
@client = klass.new
```

...into an ActiveJob Concerns file, [as described here](http://stackoverflow.com/questions/35305960/dry-in-activejobbase)?  That way we aren't repeating it in the three other Jobs.